jobs/newrelic-monitor/spec

```diff
@@ -15,3 +15,6 @@ properties:
     description: "Newrelic License Key"
   newrelic.deployment_tag:
     description: "The deployment tag that would appear in New Relic."
+  newrelic.proxy:
+    description: "http proxy to use to access newrelic saas "
```

Comment:
since http proxy environment variables format vary, suggesting to add an [example](https://bosh.io/docs/jobs.html#spec) for the supported format

example: user:password@hostname:port